src/main/java/com/zendesk/maxwell/util/InterpolatedStringsHandler.java

```diff
@@ -0,0 +1,81 @@
+package com.zendesk.maxwell.util;
+
+import com.zendesk.maxwell.row.RowIdentity;
+import com.zendesk.maxwell.row.RowMap;
+
+/**
+ * An utility class to interpolate subscription, channel templates.
+ * <p>
+ * An input string can contains any combination of:
+ * - %{db}
+ * - %{table}
+ * - %{type}
+ */
+public class InterpolatedStringsHandler {
```

Comment:
can we rename this class something a bit better?  `TopicInterpolator` would be fine.

Reply:
updated in https://github.com/zendesk/maxwell/pull/1646/commits/37ff1c30f8e41c6995e46ed0ee248dc2453d4289